.ci/pytorch/build.sh

```diff
@@ -309,3 +309,6 @@ if [[ "$BUILD_ENVIRONMENT" != *libtorch* && "$BUILD_ENVIRONMENT" != *bazel* ]];
 fi
 
 print_sccache_stats
+
```

Comment:
This code wasnt reverted properly.
Can keep it or delete. it should be harmless code.

Reply:
Thanks, I deleted it to minimize diffs vs upstream.